subsys/net/lib/Kbuild

```diff
@@ -1,3 +1,4 @@
+obj-$(CONFIG_NET_SOCKETS) += sockets/
```

Comment:
I'm not sure, why? There're dns, http on the same level, not zdns, zhttp. (There's zoap, yeah, but it's an exception.)